### PR TITLE
Enable connecting existing Commerce Accounts

### PR DESCRIPTION
**Pull request by denniskwon**

### CODE REVIEW

**Review by wvega — CHANGES_REQUESTED** [review]
@denniskwon if I understand correctly, the purpose of this PR is to:

- Show extra information about the Commerce connection in Instagram Checkout tab
- Bypass the connection onboarding if we already have a Commerce Manager ID from the page settings

My Facebook account is not setup to test the Commerce flow, but through code review and some mock tests I think the changes here achieve the items above.

I left some questions and suggestions below:

**Comment by wvega** [comment]
I'm going to cherry pick the commits from this PR and from https://github.com/facebookincubator/facebook-for-woocommerce/pull/1727#pullrequestreview-550646960 to generate a test zip file.

We should probably rebase your commits on this branch on top of the most recent `release/instagram-checkout`. I rebased `release/instagram-checkout` some time ago and now this PR shows commits that are already there, but with a different changes.

File: includes/Admin/Settings_Screens/Commerce.php [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+		} else {
+
+			$connect_url = facebook_for_woocommerce()->get_connection_handler()->get_connect_url( true );
+		}
+
+		?>
+
+		<h2><?php esc_html_e( 'Instagram Checkout', 'facebook-for-woocommerce' ); ?></h2>
+
+		<table class="form-table">
```
[the remark this answers is not in the document]
Comment (denniskwon) [reply includes/Admin/Settings_Screens/Commerce.php:212]: I believe the first conditional in the render method would take care of this, no?

[the remark this answers is not in the document]
Comment (wvega) [reply includes/Admin/Settings_Screens/Commerce.php:212]: I meant to render the table inside this if statement:

https://github.com/facebookincubator/facebook-for-woocommerce/blob/c7b3dbd2ee2b426d53a91d3f64785aa2ea5e6c8c/includes/Admin/Settings_Screens/Commerce.php#L289

I believe it's possible for users to be connected to Facebook and not have a Commerce connection yet, for example if they upgraded from a version that didn't request the page access token.

**Pull request merged**